docs/dam/integration_with_dxp.md

```diff
@@ -0,0 +1,180 @@
+---
+description: Integrate DAM with your Ibexa DXP project to use an effective tool for managing marketing assets.
+---
+
+
+# Integrate DAM with DXP
+
+DAM integration with [[= product_name =]] is dedicated to [[= product_name_base =]] users who also use Actito services. It enables using assets repository stored in the DXP in the Actito platform to build and insert marketing assets into campaign emails.
+For more information, see [Actito documentation](https://cdn3.actito.com/fe/actito-documentation/docs/Using_Assets_from_a_DAM/#connection-to-ibexa).
```

Comment:
I'm not sure about this. Integration with Actito has dedicated instance ibexa/dam. We can integrate DAM with regular DXP instance every time when we want to use DXP as data storage. Integration with Actito is irrelevant here. Correct me if I'm wrong @dkorostynski.